docs/releasing.md

```diff
@@ -61,14 +61,6 @@ cd ../lighthouse-pristine
 yarn
 yarn build-all
 
-# Verify the viewer will work.
-yarn serve-viewer
-# Works with v4 report? http://localhost:7333/viewer?gist=7251f9eba409f385e4c0424515fe8009
-# Works with v5 report? http://localhost:7333/viewer?gist=6093e41b9b50c8d642a7e6bbc784e32f
-# Works with v6 report? http://localhost:7333/viewer?gist=94722e917a507feb5371ad51be6c3334
-# Works with v8 report? http://localhost:7333/viewer?gist=18d523b86779185ecfd376d58f891e1d
-# Current production viewer (https://googlechrome.github.io/lighthouse/viewer/) has forward compat with next major LHR?
-
 # Confirm DevTools integration will work: Do some manual testing on a number of sites.
 yarn test-devtools
 yarn open-devtools
```

Comment:
If you're willing to expand the score of this PR, I don't think we need to test DevTools here either.

`yarn test-devtools` is run in CI and the "manual testing on a number of sites" part is covered by the DevTools smoke tests IMO. Maybe this entire section could be removed?